macOS build fails: objc_msgSend

```
$ make
[...]
[ 48%] Building CXX object CMakeFiles/teeworlds_srv.dir/src/game/server/eventhandler.cpp.o
[ 48%] Building CXX object CMakeFiles/teeworlds.dir/src/engine/client/client.cpp.o
[ 49%] Building CXX object CMakeFiles/teeworlds.dir/src/engine/client/friends.cpp.o
In file included from /Users/chillerdragon/Desktop/git/teeworlds/src/engine/client/backend_sdl.cpp:13:
/Users/chillerdragon/Desktop/git/teeworlds/src/engine/client/backend_sdl.h:29:4: error: no matching function for call to
      'objc_msgSend'
                        objc_msgSend(m_Pool, selector);
                        ^~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/include/objc/message.h:63:1: note: 
      candidate function not viable: requires 0 arguments, but 2 were provided
objc_msgSend(void /* id self, SEL op, ... */ )
^
In file included from /Users/chillerdragon/Desktop/git/teeworlds/src/engine/client/backend_sdl.cpp:13:
/Users/chillerdragon/Desktop/git/teeworlds/src/engine/client/backend_sdl.h:35:4: error: no matching function for call to
      'objc_msgSend'
                        objc_msgSend(m_Pool, selector);
                        ^~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/include/objc/message.h:63:1: note: 
      candidate function not viable: requires 0 arguments, but 2 were provided
objc_msgSend(void /* id self, SEL op, ... */ )
^
2 errors generated.
make[2]: *** [CMakeFiles/teeworlds.dir/src/engine/client/backend_sdl.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 50%] Building CXX object CMakeFiles/teeworlds_srv.dir/src/game/server/gamecontroller.cpp.o
[ 50%] Building CXX object CMakeFiles/teeworlds_srv.dir/src/game/server/gamemodes/ctf.cpp.o
```
Might be a useful resource: https://github.com/bitcoin/bitcoin/pull/16720

--- a/src/engine/client/backend_sdl.h
+++ b/src/engine/client/backend_sdl.h
@@ -26,13 +26,13 @@
 			Class NSAutoreleasePoolClass = (Class) objc_getClass("NSAutoreleasePool");
 			m_Pool = class_createInstance(NSAutoreleasePoolClass, 0);
 			SEL selector = sel_registerName("init");
-			objc_msgSend(m_Pool, selector);
+			((id (*)(id, SEL))objc_msgSend)(m_Pool, selector);
 		}
 
 		~CAutoreleasePool()
 		{
 			SEL selector = sel_registerName("drain");
-			objc_msgSend(m_Pool, selector);
+			((id (*)(id, SEL))objc_msgSend)(m_Pool, selector);
 		}
 	};
 #endif

